course/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+# import user
+
+# All the comments are to be changed after the PR merge of User and Teacher_course models
+
+
+class ReviewManager(models.Manager):
+
+    def get_reviews_by_course(self, course_id: int):
+        reviews = self.filter(course_id=course_id)
+        return reviews
+
+    def get_avg_rating_by_course(self, course_id: int):
+        avg_rating = self.filter(course_id=course_id).aggregate(models.Avg('rating'))['rating__avg']
+        return avg_rating
+
+
+class Review(models.Model):
+    class Rating(models.IntegerChoices):
+        ONE_STAR = 1
+        TWO_STARS = 2
+        THREE_STARS = 3
+        FOUR_STARS = 4
+        FIVE_STARS = 5
+
+    review_id = models.BigAutoField(primary_key=True)
+    # student_id = models.ForeignKey(user, on_delete=models.CASCADE)
+    # course_id = models.ForeignKey(Teacher_course, on_delete=models.CASCADE)
+    rating = models.IntegerField(choices=Rating.choices)
```

Comment:
I think this will allow multiple choices for the rating. What do you think about something like this?
```suggestion
    rating = models.IntegerField(validators=[MaxValueValidator(5),MinValueValidator(1)])
```

Reply:
> I think this will allow multiple choices for the rating. What do you think about something like this?

Yes, I think it's more correct in this way, I fixed it.